package.json

```diff
@@ -62,6 +58,7 @@
     "chai": "3.5.0",
     "conventional-changelog": "^1.1.0",
     "css-loader": "0.26.1",
+    "del": "^2.2.2",
```

Comment:
Maybe use exact numbers ? Thanks to my current update nightmare issues, it might be better to stick to exact numbers in general